internal/service/networkfirewall/firewall_policy.go

```diff
@@ -438,6 +447,10 @@ func flattenPolicyStatefulRuleGroupReference(l []*networkfirewall.StatefulRuleGr
 		if ref.Priority != nil {
 			reference["priority"] = int(aws.Int64Value(ref.Priority))
 		}
+		if ref.Override != nil {
+			reference["override_action"] = aws.StringValue(ref.Override.Action)
+		}
+
```

Comment:
Similarly for the read, we will use a `flatten` function to extract `action` from `override`

Reply:
Updated.